go/driver_eg.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package main
+
+import "C"
+import (
+	"fmt"
+	"typedb_driver/go_wrapper"
+)
+
+func openCoreFunc() {
+	// Test that we can call functions using the implementation in the dynamic library.
+	value := 667
```

Comment:
Is it a code convention for `go` to have that long tabs?
Btw, it's not even spaces, its like a huge tab symbol! `	`
It's a **really good** practice to use spaces instead.
I am pretty sure that `Go` has a similar setting in IntelliJ:
![image](https://github.com/vaticle/typedb-driver/assets/22564079/21d74771-606b-471d-9243-1851f92a2455)